python27_sti_base/Makefile

```diff
@@ -0,0 +1,46 @@
+BASE_IMAGE=centos/python-27-centos7:latest
+LOCAL_IMAGE=openshift-spark-py27i-sti
+SPARK_IMAGE=myproject/openshift-spark-py27-sti
+
+.PHONY: build clean push create destroy artifacts clean-artifacts clean-dockerfile clean-scripts clean-template
+
+build:
+	docker build -t $(LOCAL_IMAGE) .
```

Comment:
given the recent discussion about docker, should we be adding a sudo or some wrapper script here?